helm/alfresco-content-services/templates/deployment-transform-router.yaml

```diff
@@ -1,5 +1,5 @@
 apiVersion: extensions/v1beta1
-kind: Deployment
+apiVersion: apps/v1
```

Comment:
Is this a typo? I presume we need to keep the "kind: Deployment" and change the previous line!?

Reply:
yes definitely